org-roam-graph.el

```diff
@@ -186,18 +186,22 @@ If PREFIX, then the graph is generated but the viewer is not invoked."
 	        (call-process org-roam-graph-viewer nil 0 nil temp-graph)
         (view-file temp-graph)))))
 
-(defun org-roam-graph-show-connected-component (&optional prefix)
+(defun org-roam-graph-show-connected-component (&optional max-distance no-display)
```

Comment:
I'm not too happy about breaking the existing interface here, so maybe this should be moved to a separate function now. (I just started this experiment by extending the existing "connected-component" function.)
Previously the prefix was used to suppress showing of the graph. But I found that a numeric prefix was a very convenient way to set the maximum distance, so I changed that behavior…

Reply:
I think breakage now is still fine, since this functionality was only recently merged, shouldn't affect many.

I do think it's a bit odd for a function called `org-roam-graph-show`, to not actually show anything, the building logic can probably be extracted away from the showing logic.

Reply:
I like the idea of having two functions instead of using the prefix to control whether the graph is shown or not. :+1: The numerical prefix could then be used to indicate the maximum distance for both the interactive "build" function and the interactive "build&show" function.

Reply:
Would it be fine for you if I then also adapted `org-roam-graph-show` to ignore the prefix and instead added a `org-roam-graph-build` interactive function?

Reply:
> Would it be fine for you if I then also adapted `org-roam-graph-show` to ignore the prefix and instead added a `org-roam-graph-build` interactive function? (Maybe also in a follow-up PR, with a separate commit on the master branch if you prefer.)

Yes, this is fine.

Reply:
Okay, then I'll create a second PR as soon as this one is merged. :slightly_smiling_face:

Reply:
→ #426, which I'll update after you hit merge on the current PR. :+1: